### ISSUE TITLE
randT & sresT seems useless

### ISSUE THREAD

**Issue by jeremad** [lesleyxyz/node-tile#5]
I re-implemented this in flutter, and I have come t the conclusion that randT and sresT are useless. Is it a misunderstanding of the protocole, or an actual security issue?

**Comment by lesleyxyz** [comment]
Hi jeremad,

randT and sresT is used in the official Tile app [here](https://github.com/lesleyxyz/node-tile/blob/main/src/services/AbstractTileService.ts#L76-L82) instead of the throw, in the case that `this.isMepCmdOrRespSet()` is false. However, I haven't had a case where it would ever be false.

Because I wasn't able to trigger it using the official app I decided to not implement it.
I believe it's linked to functionality of an older tile or something which did not have a "toa mep processor".

This is not a security issue as if there is a "mep processor" available, it generates the hmac using the authKey which is considered secure

So in conclusion, yes, at the moment randT & sresT is useless. However I kept it there in the case I would ever implement this.

**Comment by jeremad** [comment]
:bow:
